src/annotations.ts

```diff
@@ -0,0 +1,5 @@
+import { AnnotationsQueryCodeEditor } from './AnnotationsQueryCodeEditor';
+
+export const RedshiftAnnotationsSupport = {
+  QueryEditor: AnnotationsQueryCodeEditor,
```

Comment:
Added all this into separate files so it's neater for when getDefaultQuery will be added later(in Grafana 9.5)